dev-requirements.txt

```diff
@@ -1,3 +1,19 @@
-pytest==2.8.5
-pytest-mock==0.9.0
-responses
+#
+# This file is autogenerated by pip-compile
+# To update, run:
```

Comment:
This file was autogenerated, but, to answer your question: as often as necessary? Whenever gemnasium tells us to?